libexec/keepalived-check-process.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+################################################################################
+# keepalived-check-process.sh - Checks acess to a PID
+################################################################################
+#
+# Copyright (C) 2019 Adfinis SyGroup AG
+#                    https://adfinis-sygroup.ch
+#                    info@adfinis-sygroup.ch
+#
+# This program is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public
+# License as published  by the Free Software Foundation, version
+# 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License  along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Please submit enhancements, bugfixes or comments via:
+# https://github.com/adfinis-sygroup/base4kids2-keepalived-scripts
+#
+# Authors:
+#  Christian Affolter <christian.affolter@adfinis-sygroup.ch>
+#  Andreas Gruhler <andreas.gruhler@adfinis-sygroup.ch>
+#
+# Description:
+# This script is intended to be used together with Keepalived as a
+# VRRP tracking script. It checks the access to a process by means 
+# of a SIGNULL (NULL) 'signal'. No actual signal is sent but it is 
+# checked if the caller would be allowed to send a signal and if 
+# the process exists, see 'man 2 kill'.
+# 
+# It returns 0 on success or a non-zero exit status on failures.
+# 
+# The script expects the name of the process or service to perform
+# the check on to be passed by argument or environment variable.
+#
+# See also:
+# http://www.keepalived.org/
+# https://github.com/adfinis-sygroup/base4kids2-keepalived-scripts
+#
+# Usage:
+# ./keepalived-check-process.sh KEEPALIVED_CHECK_PROCESS_NAME
+#
+# See keepalived-check-process.sh -h for further options
+#
+
+# Enable pipefail:
+# The return value of a pipeline is the value of the last (rightmost) command
+# to exit with a non-zero status, or zero if all commands in the pipeline exit
+# successfully.
+set -o pipefail
+
```

Comment:
Please re-add the external command check for the `killall` command, see https://github.com/adfinis-sygroup/base4kids2-keepalived-scripts/blob/9f21b12166183a773b200f564f7169eb644c4567/libexec/keepalived-check-ldap.sh#L58-L71

Reply:
This part has been added again, including the `killall` command